utils/terrascript_client.py

```diff
@@ -555,15 +555,15 @@ def populate_tf_resource_rds(self, resource, namespace_info,
             deps = [pg_tf_resource]
             values['parameter_group_name'] = pg_name
 
-        enhanced_monitoring = values.pop('enhanced_monitoring')
+        enhanced_monitoring = values.pop('enhanced_monitoring', None)
 
         # monitoring interval should only be set if enhanced monitoring
         # is true
         if (
             not enhanced_monitoring and
             values.get('monitoring_interval', None)
         ):
-            values.pop('monitoring_interval')
+            values.pop('monitoring_interval', None)
```

Comment:
Do you need this? You've already checked `values.get('monitoring_interval')` is true

Reply:
doesn't hurt?

Reply:
It doesn't, just redundant :)